### ISSUE TITLE
Rename this mod

### ISSUE THREAD

**Issue by andrewminer** [oldjunyi/NoMobSpawningOnTrees#11]
I was hunting around for a mod which allows me to relegate mobs to only spawn underground, but which wasn't super, super complicated (e.g., CustomMobSpawner).  I only gave this one a passing glance (because spawning on trees doesn't bother me too much).  It was only after I read through the wiki and figured out *just how powerful* this mod is that I got really excited.  This is the mob spawning mod I've wanted... *way* better the CustomMobSpawner.  The rules are simple, clear, and yet extremely expressive.  Perfect!

The only problem is that no one would know it from the name!  It makes it sound like a much more limited mod than it really is, and I suspect, makes people pass it by when shopping about for mob control mods.  Here's a list of words which may work well in a new name:

    Total       Spawn      Control
    Complete    Mob        Mastery
    Ultimate    Entity     Tuner
    Perfect     Spawning   Tweaker
                           Overrides

I'm sure there are lots of other good names out there, but I'd love to see this mod get a name which helps people understand how powerful and general-purpose it is.

**Comment by WolfieWaffle** [comment]
I agree, I thought this mod only prevented them on trees until I read full description :P
